README.md

```diff
@@ -356,7 +356,7 @@ To enable security of specific provider, please read the sections below.
 ### Management Portal
 To enable security via the [RADAR Management Portal](https://github.com/RADAR-base/ManagementPortal), set the following property -
 ```ini
-security.radar.managementportal.url=true
+security.radar.managementportal.enabled=true
```

Comment:
Thanks for the correction. Can you please also add the configuration for the URL is needed to point to the correct MP instance -
```ini
security.radar.managementportal.url=<your management portal base url>
```

Reply:
👍 Done.